platforms/qrb2210-unoq/emmc/partitions_user_16G.conf

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2025 Qualcomm Innovation Center, Inc. All rights reserved.
```

Comment:
Just to note, this file be ignored. If it's something that you want to use in your builds, it is easier to spawn the 'qrb2210-unoq-16GiB' device.